src/views/planningPoker.tsx

```diff
@@ -199,6 +199,7 @@ const PlanningPoker = ({ record, initialVotes }) => {
               >
                 See results
               </button>
+              <p className="votes">{votes.length} Votes</p>
```

Comment:
Thanks for the contribution @NGromann! What do you think about combining the vote count with the "See results" button right above? (e.g. "See X votes") That would keep the UI a bit tidier.

<img width="245" alt="Screenshot 2023-03-07 at 3 51 55 PM" src="https://user-images.githubusercontent.com/104908/223549663-b379ed28-3782-45c3-845c-7d3967db7327.png">

Reply:
@trydionel Good idea! I updated it.